examples/pytorch/aim.py

```diff
@@ -65,11 +65,17 @@ def forward(self, images):
 
 transform = AIMTransform()
 # we ignore object detection annotations by setting target_transform to return 0
+
+
+def target_transform_function(t):
```

Comment:
Can we call this just `target_transform`? It keeps the examples a bit more concise and matches the `target_transform` argument name from the dataset.

Reply:
yeah sure I will update it right now